functions/src/RSVP/checkValidID.js

```diff
@@ -9,8 +9,16 @@ exports.checkValidID = functions.https.onRequest((request, response) => {
   response.set('Access-Control-Allow-Headers', 'Content-Type, crossDomain');
 
   const { id } = JSON.parse(request.body);
+  const { applicantType } = JSON.parse(request.body);
```

Comment:
For future reference, you can store the result of `JSON.parse(request.body)` in some variable and use that instead of parsing the body twice, or use a single destructing assignment `const { id, applicantType} = JSON.parse(request.body)`.

Reply:
Good point, forgot I could do that. Just fixed!